### PR TITLE
update to v0.15

### PR DESCRIPTION
**Pull request by kalantar**
Update charts and tests

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -6,7 +6,7 @@ RUN apt-get update && apt-get install -y curl
 
 # Set Iter8 version from build args
 ARG TAG
-ENV TAG=${TAG:-v0.14.10}
+ENV TAG=${TAG:-v0.15.1}
```
Comment (Alan-Cha) [inline Dockerfile]: I notice that this is `v0.15.1`...

File: charts/iter8/Chart.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: iter8
-version: 0.14.3
+version: 0.15.0
```
Comment (Alan-Cha) [inline charts/iter8/Chart.yaml:3]: ... and this is `v0.15.0`. Is this what we want?

File: Dockerfile (hunk 1 above)
Comment (kalantar) [reply Dockerfile]: fixed

File: charts/iter8/Chart.yaml (hunk 2 above)
Comment (kalantar) [reply charts/iter8/Chart.yaml:3]: we want this one

**Review by Alan-Cha — APPROVED** [review]

**Pull request merged**